pages/resources/[page].js

```diff
@@ -64,7 +64,7 @@ function Resources() {
 
   const handleEndpoint = () => {
     if (q) {
-      return getResourcesBySearch({ page, category, languages, paid, q });
+      return getResourcesBySearch({ page: page - 1, category, languages, paid, q });
     }
     return getResourcesPromise({ page, category, languages, paid });
   };
```

Comment:
Shouldn't it also be on line 69?

Reply:
That is the regular resources endpoint so it isn’t 0 indexed like the search endpoint.